Formula/h/hashlink.rb

```diff
@@ -46,17 +46,35 @@ class Hashlink < Formula
   end
 
   def install
+    # NOTE: This installs lib/*.hdll files which would be audited by `--new`.
+    # These appear to be renamed shared libraries specifically used by HashLink.
+    args = ["PREFIX=#{prefix}"]
```

Comment:
We could add `.hdll` as a valid extension at https://github.com/Homebrew/brew/blob/81c4351fb50a1906f6f73411c41d507a5da3fe5d/Library/Homebrew/formula_cellar_checks.rb#L81

Though nothing currently depends on HashLink so mainly considering this as an exception.